web/index.js

```diff
@@ -0,0 +1,31 @@
+const firebase=require('firebase');
+const express=require('express');
+const app=express();
+const hbs=require('hbs');
+const port=process.env.PORT || 3000;
+
+var config = {
```

Comment:
We canno tpush the configs here on github. Otherwise someone else may misuse your keys. Please add it to new config file and link it here. Also add the file in gitgnore